common/app/model/content.scala

```diff
@@ -175,15 +175,15 @@ final case class Content(
 
   lazy val linkCounts = LinkTo.countLinks(fields.body) + fields.standfirst.map(LinkTo.countLinks).getOrElse(LinkCounts.None)
 
-  lazy val hasMultipleVideosInPage: Boolean = mainVideoCanonicalPath match {
-    case Some(_) => numberOfVideosInTheBody > 0
-    case None => numberOfVideosInTheBody > 1
-  }
+  lazy val mainMediaVideo = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video")
 
-  lazy val mainVideoCanonicalPath: Option[String] = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video").headOption.map { v =>
-    new URL(v.attr("data-canonical-url")).getPath.stripPrefix("/")
+  lazy val hasMultipleVideosInPage: Boolean = mainMediaVideo.isEmpty match {
+    case true   => numberOfVideosInTheBody > 1
+    case false  => numberOfVideosInTheBody > 0
```

Comment:
Don't think that'll work. The (rather convoluted) logic is counting the number of videos in an article:

if there is one in main media, then check if the body has at least 1 (so total is at least 2)
if there isn't one in main media, then check if the body has more than 1 video